crates/red_knot_python_semantic/resources/mdtest/properties.md

```diff
@@ -203,7 +203,7 @@ reveal_type(c.attr)  # revealed: Unknown
 In this section, we trace through some of the steps that make properties work. We start with a
```

Comment:
You need to delete the big TODO comment immediately prior to this line as well ;)